modules/pensions/spec/models/saved_claim/pension_spec.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'spec_helper'
+require_relative '../../support/saved_claims_spec_helper'
+
+RSpec.describe Pensions::SavedClaim::Pension, uploader_helpers: true do
+  subject { described_class.new }
+
+  let(:instance) { FactoryBot.build(:pensions_module_pension_claim) }
+
+  it_behaves_like 'saved_claim_with_confirmation_number'
+
+  context 'saved claims w/ attachments' do
+    stub_virus_scan
+
+    let!(:attachment1) { FactoryBot.create(:pensions_module_pension_burial) }
+    let!(:attachment2) { FactoryBot.create(:pensions_module_pension_burial) }
+
+    let(:claim) do
+      FactoryBot.create(
+        :pensions_module_pension_claim,
+        form: {
+          veteranFullName: {
+            first: 'Test',
+            last: 'User'
+          },
+          email: 'foo@foo.com',
+          veteranDateOfBirth: '1989-12-13',
+          veteranSocialSecurityNumber: '111223333',
+          files: [
+            {
+              confirmationCode: attachment1.guid
+            },
+            {
+              confirmationCode: attachment2.guid
+            }
+          ],
+          veteranAddress: {
+            country: 'USA',
+            state: 'CA',
+            postalCode: '90210',
+            street: '123 Main St',
+            city: 'Anytown'
+          },
+          statementOfTruthCertified: true,
+          statementOfTruthSignature: 'Test User'
+        }.to_json
+      )
+    end
+
+    describe '#process_attachments!' do
+      it 'sets the attachments saved_claim_id' do
+        expect(Pensions::Lighthouse::SubmitBenefitsIntakeClaim).to receive(:perform_async).with(claim.id)
+        claim.process_attachments!
+        expect(claim.persistent_attachments.size).to eq(2)
+      end
+    end
```

Comment:
I would like to see a move towards a 'submit' function on SavedClaim which would in turn call a registered sidekiq job for the specific class or a default.  This would/should be separate from relating the attachments ... High impact changes that are out of scope for this.